docs/redoc/v1.9.x/openapi.json

```diff
@@ -0,0 +1,11035 @@
+    "contact": {
+      "email": "andrey@vasnetsov.com"
+    },
+    "license": {
+      "name": "Apache 2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    },
+    "version": "master"
```

Comment:
```suggestion
    "version": "v1.9.x"
```

Reply:
Good catch! I'll put it in my notes so I don't forget next time.